dask/dot.py

```diff
@@ -292,7 +293,7 @@ def graphviz_to_file(g, filename, format):
 
     display_cls = _get_display_cls(format)
 
-    if not filename:
+    if filename is None:
         return display_cls(data=data)
```

Comment:
You could use `monkeypatch` to temporarily change `display_cls`, so that it has an effect you can check from the test function.

Do you know how to do this?

Reply:
No, never heard of `monkeypatch`. Can you point me to relevant resources?

Reply:
I went through [this](https://docs.pytest.org/en/6.2.x/monkeypatch.html) and this is exactly what I was looking for! Wow :100:

Reply:
Ping me if you cannot quickly figure it out, I can show you how I would do this.